### ISSUE TITLE
Can send trucks to train depot

### ISSUE THREAD

**Issue by Condoamanti** [voxeltycoon/bug-tracker#228]
**Describe the bug**
Currently if you send a truck to garage, but instead select the depot window for trains, the trucks say going to garage, but actually disappear.

**Steps to reproduce**
1. Select truck
2. Select 'send to garage'
3. Select 'train depot window'
4. cry

**Build version**
0.85.0.24 steam

**Platform**
Windows 10

**Reproducibility**
every time

**Logs/saves**
```
InvalidOperationException: Can't add vehicle to depot, wrong type.
  at VoxelTycoon.Tracks.VehicleDepot.AddVehicle (VoxelTycoon.Tracks.Vehicle vehicle, System.Boolean supressEvents) [0x0005b] in <3211e969e9dc48fc95fa6ef826fa17da>:0 
  at VoxelTycoon.Tracks.Tasks.GoToDepotDelayedAction.DoDelayedAction () [0x00029] in <3211e969e9dc48fc95fa6ef826fa17da>:0 
  at VoxelTycoon.Tracks.Tasks.VehicleTaskDelayedAction+<Run>d__9.MoveNext () [0x000b2] in <3211e969e9dc48fc95fa6ef826fa17da>:0 
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00026] in <eaed1887017646508aa3fd7a8edf50d6>:0
```

**Comment by Condoamanti** [comment]
![image](https://user-images.githubusercontent.com/18709199/115496311-fd7daf80-a22e-11eb-919a-cd90476bab64.png)
Now getting this after leaving the game after this error...

**Comment by andrewpey** [comment]
Fixed in [0.85.1](https://voxeltycoon.xyz/changelog#0.85.1).
